velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -84,6 +84,16 @@ class HiveConnectorTestBase : public OperatorTestBase {
     return makeHiveConnectorSplit(filePath, {}, start, length);
   }
 
+  // Split file at a given path 'filePath' in range [start, length] into
+  // 'numSplits' splits.
+  static std::vector<std::shared_ptr<connector::hive::HiveConnectorSplit>>
+  makeHiveSplitsByCount(
```

Comment:
A few observations and suggestions:
-  Other similar methods are named makeConnectorSplit. It would be nice to rename this method to be consistent. 
- This method relies on length to be the actual size of the file, while other methods accept arbitrary large lengths. Given that the usage of this method is to always pass zero for start and file size for length, it might be helpful to change the method signature to remove start and length parameters. 
- The default value for splitCount parameter seems arbitrary. Consider removing it and requiring the caller to specify number of splits.

```
static std::vector<std::shared_ptr<connector::hive::HiveConnectorSplit>>
makeHiveConnectorSplits(
   const std::string& filePath, 
   uint32_t splitCount, 
   dwio::common::FileFormat format);
```